### PR TITLE
2066 support types for new operations in lima

### PR DESCRIPTION
**Pull request by hui-an-yang**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito-rpc/test/taquito-rpc.spec.ts [hunk 1]
```diff
@@ -3211,6 +3214,86 @@ describe('RpcClient test', () => {
 
       done();
     });
+
+    it('should be able to access the properties of internal operation type event, proto15', async (done) => {
```
Comment (roxaneletourneau) [inline packages/taquito-rpc/test/taquito-rpc.spec.ts:3218]: wrong test title

File: packages/taquito-rpc/test/taquito-rpc.spec.ts [hunk 2]
```diff
@@ -3211,6 +3214,86 @@ describe('RpcClient test', () => {
...
+      expect(content.metadata.balance_updates![3].contract).toEqual(
+        'tz1hoyMUiJYYr4FRPMU8Z7WJzYkqgjygjaTy'
+      );
+      expect(content.metadata.balance_updates![3].change).toEqual('156825984');
+      expect(content.metadata.balance_updates![3].origin).toEqual('block');
+
+      done();
+    });
+
+    it('should be able to access the properties of internal operation type event, proto15', async (done) => {
```
Comment (roxaneletourneau) [inline packages/taquito-rpc/test/taquito-rpc.spec.ts:3263]: wrong test title

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]
minor comments, lgtm otherwise :)

File: packages/taquito-rpc/test/taquito-rpc.spec.ts (hunk 1 above)
Comment (hui-an-yang) [reply packages/taquito-rpc/test/taquito-rpc.spec.ts]: addressed in a new commit

File: packages/taquito-rpc/test/taquito-rpc.spec.ts (hunk 2 above)
Comment (hui-an-yang) [reply packages/taquito-rpc/test/taquito-rpc.spec.ts]: addressed in a new commit

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**